test/functional/rpc_fundrawtransaction.py

```diff
@@ -379,7 +379,7 @@ def test_fee_p2pkh(self):
 
         # Create same transaction over sendtoaddress.
         txId = self.nodes[0].sendtoaddress(self.nodes[1].getnewaddress(), 1.1)
-        signedFee = self.nodes[0].getrawmempool(True)[txId]['fee']
+        signedFee = self.nodes[0].getrawmempool(True)[txId]['fees']['base']
```

Comment:
To make the commits atomic, rpc_fundrawtransaction.py needs to be updated in 5adcba2562 as well

Reply:
ah hm, I must have fetched before you pushed.

> just to make sure i understand, the changes to rpc_fundrawtransaction.py and mempool_packages.py should be in the same commit as the changes to blockchain.cpp. this ensures the tests still pass for this commit, correct?

correct, yes :)